pinot-spi/src/main/java/org/apache/pinot/spi/ingestion/batch/BatchConfigProperties.java

```diff
@@ -43,7 +43,9 @@ private BatchConfigProperties() {
   public static final String TABLE_CONFIGS_URI = "tableConfigsURI";
   public static final String SCHEMA = "schema";
   public static final String SCHEMA_URI = "schemaURI";
+  public static final String PARTITION_ID = "partitionId";
   public static final String SEQUENCE_ID = "sequenceId";
+  public static final String SEGMENT_CREATION_TIME_MS = "segmentCreationTimeMs";
```

Comment:
nit: we should name this as `segmentUploadTimeMs` too for consistency across code.